android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlBroadcastReceiver.java

```diff
@@ -546,21 +561,21 @@ private static boolean isBluetoothConnected() {
     }
 
 
-    private static SdlDeviceListener getSdlDeviceListener(Context context, BluetoothDevice bluetoothDevice) {
+    private static SdlDeviceListener getSdlDeviceListener(Context context, BluetoothDevice bluetoothDevice, VehicleType vehicleType) {
```

Comment:
@kostyaBoss On line 564 you can remove the parameter `VehicleType vehicleType` from the method signature.  `getSdlDeviceListener` will then have 2 parameters which will match with the proposal.